cdap-validation/bin/testing.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/python
+
+import os
+import test_helpers as helpers
+import json
+import re
+from subprocess import call
+import ambari
+import cm
+
+##### TEST FRAMEWORK FUNCTIONS #####
+
+input_json = 'input.json'
+
+# calls all other functions
+def testing(base_vars,cluster_vars):
+    if cluster_vars['verbose'] == 2: print 'start testing'
+
+    # prep input variables
+    #cluster_vars = {}
+    input = read_input(input_json)
+    cluster_vars['host'] = input['params']['uri']
+    cluster_vars['username'],cluster_vars['password']= input['params']['userpass'].split(':')
+    cluster_vars['cluster'] = input['params']['cluster_name']
```

Comment:
why do these use different key names? Otherwise you could just overwrite whats in cluster_vars with everything in input params